src/js/components/TaskDetailComponent.jsx

```diff
@@ -27,7 +27,12 @@ var TaskDetailComponent = React.createClass({
     return (
       <p className="text-center text-danger">
         Error fetching task details.
-        Go to <Link to="app" params={{appId: encodeURIComponent(this.props.appId)}}>Task List</Link> to see the full list.
+        Go to
+        <Link to="app"
+            params={{appId: encodeURIComponent(this.props.appId)}}>
+          Task List
+        </Link>
+        to see the full list.
```

Comment:
This renders like so:

![screen shot 2015-09-02 at 14 46 03](https://cloud.githubusercontent.com/assets/1078545/9631476/5fecfb94-5181-11e5-81b0-4637b5f9dae9.png)